Plan/common/src/main/resources/assets/plan/locale/locale_CN.yml

```diff
@@ -411,24 +411,24 @@ html:
             testResult: "测试结果"
             threshold: "阈值"
             thresholdUnit: "小时/周"
-            tips: "Tips"
+            tips: "提示"
             usingTheGraph: "使用图表"
         hourByHour: "按小时查看"
         inactive: "不活跃"
-        indexInactive: "不活跃"
-        indexRegular: "经常上线"
+        在dexInactive: "不活跃"
```

Comment:
Keys can't be translated. There are multiple places where find-and-replace has changed keys in the file. Please fix them.

Reply:
Right away

Reply:
This should be caused by the browser plugin